src/osemgrep/core/Metrics_.ml

```diff
@@ -286,7 +286,7 @@ let add_engine_type (engine_type : Engine_type.t) =
             | Intraprocedural -> `Intraprocedural
             | Interprocedural -> `Interprocedural
             | Interfile -> `Interfile);
-          code_config;
+          code_config = Option.map (Fun.const true) code_config;
```

Comment:
This is not readable. Avoid Option.map and Fun.xxx when you can and when the resulting simpler code is actually shorter.

Reply:
Do you find the version in 00fcd33e6b5dbebb7123834e4863fb70c6a875ca objectionable? I can't tell from this if you are opposed to the combination or both `Option.map` and `Fun` individually.

Reply:
I usually find code more readable when they don't use Fun.const, or the $ operator, or curry, or Option.value, etc. KISS. In this case I think `match code_config with None -> false |  Some _ -> true' more readable.